src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala

```diff
@@ -265,6 +270,10 @@ object FeeSchedule {
     override val G_cold_sload = 2100
     override val G_cold_account_access = 2600
     override val G_warm_storage_read = 100
+
+    // note: the access list does not exist until magneto hard fork
```

Comment:
in fact this is also true for the values just above (up to `G_cold_sload`) so we can move that comment up.